### PR TITLE
Added functionality to TopBar.

### PR DESCRIPTION
**Pull request by burkegg**

### CODE REVIEW

**Review by shaneyuan18 — APPROVED** [review]
Amazing stuff. 
One thing that I notice is that you use red vote and grey vote as array for rendering stars. I know you did this to use map, but I think this might take extra memories that you do not need.

File: client/ReviewModule.jsx [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
     this.handleTextChange = this.handleTextChange.bind(this);
+    this.handleStarHover = this.handleStarHover.bind(this);
+    this.handleMouseLeave = this.handleMouseLeave.bind(this);
   }
 
   componentDidMount() {
     this.avgStars();
-    this.fetchReviews(3);
+    const pickOne = Math.ceil(Math.random() * 100);
+    this.fetchReviews(pickOne);
```
Comment (alilthumuch) [inline client/ReviewModule.jsx]: pick0ne a little vague, maybe try restaurantID?

File: client/ReviewModule.jsx [hunk 2]
```diff
@@ -49,6 +56,30 @@ export default class ReviewModule extends React.Component {
...
+      redVote: [],
+      greyVote: [1, 2, 3, 4, 5],
+    });
+  }
+
+  handleStarHover(e) {
+    const { redVote, greyVote } = this.state;
+    e.preventDefault();
+    const starNum = parseInt(e.target.id, 10);
+    console.log(starNum);
```
Comment (alilthumuch) [inline client/ReviewModule.jsx:70]: is console.log needed?

File: client/ReviewModule.jsx [hunk 3]
```diff
@@ -69,8 +103,12 @@ export default class ReviewModule extends React.Component {
           name={name}
           handleTextChange={this.handleTextChange}
           searchText={searchText}
+          handleHover={this.handleStarHover}
```
Comment (alilthumuch) [inline client/ReviewModule.jsx]: i think Airbnb wants to set off operators with spaces. For example, handleHover = {this.handleStarHover}

File: client/ReviewModule.jsx [hunk 4]
```diff
@@ -69,8 +103,12 @@ export default class ReviewModule extends React.Component {
...
+          starVote={redVote}
```
Comment (alilthumuch) [inline client/ReviewModule.jsx]: space needed.

File: client/TopBar.jsx [hunk 5]
```diff
@@ -1,56 +1,105 @@
...
+              Language: Hipster
+              </option>
+            </select>
+          </span>
+        </div>
+        <span id="stars" onMouseLeave={handleMouseLeave}>
+          <span>
+            {starVote.map(number => (
+              <i
+                className="far fa-star fa-3x"
```
Comment (alilthumuch) [inline client/TopBar.jsx:70]: space needed before and after equal signs. [continue down]

File: client/TopBar.jsx [hunk 6]
```diff
@@ -1,56 +1,105 @@
...
+                style={{ color: 'red' }}
+                id={number.toString()}
+                onMouseOver={handleHover}
+                onFocus={handleHover}
+                key={number.toString()}
```
Comment (alilthumuch) [inline client/TopBar.jsx]: space

**Review by alilthumuch — CHANGES_REQUESTED** [review]
Looks mostly good. Main changes are the spacing around equal signs for airbnb whitespace rules for operators.

File: client/ReviewModule.jsx (hunk 2 above)
Comment (burkegg) [reply client/ReviewModule.jsx:70]: Deleted.

**Review by alilthumuch — APPROVED** [review]

**Pull request merged**